examples/experimental/dagster-airlift/dagster_airlift/in_airflow/scaffolding.py

```diff
@@ -0,0 +1,52 @@
+import os
+from pathlib import Path
+from typing import TYPE_CHECKING, Mapping
+
+import yaml
+
+if TYPE_CHECKING:
+    from airflow import DAG
+
+
+def verify_airflow_home_set() -> None:
+    if not os.getenv("AIRFLOW_HOME"):
+        raise Exception(
+            "AIRFLOW_HOME not set. Please set AIRFLOW_HOME to the root of your Airflow installation."
+        )
+
+
+def get_airflow_dags_folder() -> Path:
+    # Keep import within fxn to ensure that we perform necessary verification steps first.
+    from airflow.configuration import conf
+
+    return Path(conf.get("core", "dags_folder"))
+
+
+def get_all_dags() -> Mapping[str, "DAG"]:
+    from airflow.models import DagBag
+
+    return DagBag().dags
+
+
+def scaffold_proxied_state() -> None:
+    """Scaffolds a proxied state folder for the current Airflow installation.
+    Each proxied state is marked as False.
+    """
+    verify_airflow_home_set()
```

Comment:
should we allow airflow home to be passed as a param rather than environment?

Reply:
Probably but I'll do that in a follow up